### DISCUSSION TITLE
Return a custom dataset from select

### DISCUSSION THREAD

**Discussion by sagr4019** [uptrace/bun#224] [Q&A] [answered]
Hey, 
is it possible to return a custom dataset from a select
like in this example:
https://www.w3schools.com/sql/trysql.asp?filename=trysql_select_groupby1
I can not find anything in the docs about that

**Comment by vmihailenco** [comment, marked as answer]
See https://bun.uptrace.dev/playground/

```go
db.NewSelect().
	ColumnExpr("Shippers.ShipperName").
	ColumnExpr("COUNT(Orders.OrderID) AS NumberOfOrders").
	TableExpr("Orders").
	Join("LEFT JOIN Shippers ON Orders.ShipperID = Shippers.ShipperID").
	GroupExpr("ShipperName")
```

**Reply by sagr4019** [reply]
Thank you! Is there a link to this playground page in the documentation? It's a really good starting point to understand buns semantic

**Reply by vmihailenco** [reply]
I will add the link back.
